erpnext_quota/erpnext_quota/doctype/usage_info/usage_info.json

```diff
@@ -113,12 +116,29 @@
    "fieldtype": "Int",
    "label": "Active Company",
    "read_only": 1
+  },
+  {
+   "fieldname": "sales_invoice",
```

Comment:
Any specific reason for adding this field?

Reply:
My bad it was the initial requirement, but decided to make the limit not limited to sales invoice